apps/web/app/features/stacking/user-positions.tsx

```diff
@@ -12,9 +14,10 @@ import {
   getPoolSlugByPoolName,
 } from '~/features/stacking/start-pooled-stacking/utils/utils-preset-pools';
 import { ValueDisplayer } from '~/pages/sbtc-rewards/components/reward-value-displayer';
+import { useFakeLoading } from '~/utils/fake-loading';
```

Comment:
Can you please move `ValueDisplayer` to common area so we do not cross import from other feature's children?